tests/common/test_cgroupstelemetry.py

```diff
@@ -146,145 +245,131 @@ def test_telemetry_polling_with_active_cgroups(self, *args):
+                                                                           max_memory_usage=[current_max_memory] * data_count,
+                                                                           proc_ids=TestCGroupsTelemetry.TestProcessIds,
+                                                                           memory_statm_memory_usage=[current_proc_statm] * data_count)
+                            self.assertEqual(len(metrics), num_extensions * num_of_metrics_per_extn_expected)
+                            self._assert_polled_metrics_equal(metrics, current_cpu, current_memory, current_max_memory,
+                                                              current_proc_statm)
 
-                        patch_get_cpu_usage.return_value = current_cpu
-                        patch_get_memory_usage.return_value = current_memory  # example 200 MB
-                        patch_get_memory_max_usage.return_value = current_max_memory  # example 450 MB
+        collected_metrics = CGroupsTelemetry.report_all_tracked()
 
-                        for i in range(num_extensions):
-                            self.assertTrue(CGroupsTelemetry.is_tracked("dummy_cpu_path_{0}".format(i)))
-                            self.assertTrue(CGroupsTelemetry.is_tracked("dummy_memory_path_{0}".format(i)))
+        self._assert_extension_metrics_data(collected_metrics, num_extensions,
+                                            [current_cpu] * num_polls,
+                                            [TestCGroupsTelemetry.TestProcStatmMemoryUsed] * num_polls,
+                                            [current_memory] * num_polls,
+                                            [current_max_memory] * num_polls,
+                                            is_cpu_present=False)
 
-                        metrics = CGroupsTelemetry.poll_all_tracked()
+        self.assertEqual(CGroupsTelemetry._cgroup_metrics.__len__(), num_extensions)
+        self._assert_calculated_resource_metrics_equal([], [], [], [], [])
 
-                        for i in range(num_extensions):
-                            self.assertFalse(CGroupsTelemetry.is_tracked("dummy_cpu_path_{0}".format(i)))
-                            self.assertFalse(CGroupsTelemetry.is_tracked("dummy_memory_path_{0}".format(i)))
-
-                        self.assertEqual(CGroupsTelemetry._cgroup_metrics.__len__(), num_extensions)
-                        self._assert_cgroup_metrics_equal(
-                            cpu_usage=[current_cpu] * data_count,
-                            memory_usage=[current_memory] * data_count,
-                            max_memory_usage=[current_max_memory] * data_count)
-                        self.assertEqual(len(metrics), num_extensions * 3)
-                        self._assert_cgroup_polling_metrics_equal(metrics, current_cpu, current_memory, current_max_memory)
+    @patch("azurelinuxagent.common.osutil.default.DefaultOSUtil.get_total_cpu_ticks_since_boot")
```

Comment:
why is this mock needed?

Reply:
Stray mock left, during solving the merge conflicts while merging `develop`. Thanks. Fixed.